src/main/resources/db/local/V99_0__draft_referrals.sql

```diff
@@ -3,3 +3,6 @@ values ('ac386c25-52c8-41fa-9213-fcf42e24b0b5', '2020-12-07 18:02:01.599803+00',
        ('dfb64747-f658-40e0-a827-87b4b0bdcfed', '2020-12-07 20:45:21.986389+00', '2021-03-01', '8751622134', 'delius', null),
        ('d496e4a7-7cc1-44ea-ba67-c295084f1962', '2020-12-24 09:32:32.871623+00', '2021-01-30', '2500128586', 'delius', '428ee70f-3001-4399-95a6-ad25eaaede16'),
        ('1219a064-709b-4b6c-a11e-10b8cb3966f6', '2021-01-12 14:46:21.987234+00', null, '2500128586', 'delius', '428ee70f-3001-4399-95a6-ad25eaaede16');
+
+insert into service_user (referral_id, address, crn, disabilities, dob, ethnicity, title, first_name, last_name, needs, nomis_number, other_names, pnc_number, preferred_language, religion_or_belief, sex, sexual_orientation)
```

Comment:
If we intend this to be 1-to-1 can we choose another name, please? Based on the name, I'd assume it's a primary table. What do you think about `referral_service_user_snapshot`?

Reply:
sure thing

Reply:
change to `referral_service_user_data`